userguide/object_detection/export-coreml.md

```diff
@@ -25,17 +25,18 @@ let objectRecognition = VNCoreMLRequest(model: visionModel,
     for case let foundObject as VNRecognizedObjectObservation in results {
         let bestLabel = foundObject.labels.first! // Label with highest confidence
         let objectBounds = foundObject.boundingBox
+        let confidence = foundObject.confidence // Confidence for the predicted class
+        // bestLabel.confidence is the normalized score across all classes, which might be misleading.
```

Comment:
Given the bug you're fixing, I understand why you mention this here, but perhaps it would be cleaner for our users just to omit this comment.

Reply:
Just deleted the comment .